ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -322,6 +322,15 @@ open class SideTabBar: UIView {
         return nil
     }
 
+    /// This method returns an optional view for the given tab bar item.
+    /// It will searche the top section first and then the bottom one if not found in the top section.
```

Comment:
typo: search instead of "searche"

How about phrasing it like the below?

"Searches for the view in the top and bottom sections in that order of priority. Returns nil if the view is not found in either section."